packages/components/src/dialog/index.tsx

```diff
@@ -81,6 +83,7 @@ const Dialog = ( {
 			role="dialog"
 			shouldCloseOnEsc={ shouldCloseOnEsc }
 			shouldCloseOnOverlayClick={ shouldCloseOnOverlayClick }
+			{ ...( bodyOpenClassName ? { bodyOpenClassName } : {} ) }
```

Comment:
In the [documentation](https://reactcommunity.org/react-modal/styles/classes/#for-the-documentbody-and-html-tag), it states, "bodyOpenClassName when set as null doesn't add any class to document.body." So, for backward compatibility, we don't add the prop to the Modal component if it hasn't been passed to us.

Reply:
This seems safe. 

Would just adding the prop as `bodyOpenClassName={ bodyOpenClassName }` be just as safe maybe? I suppose it would supply a null or falsy value in the same situations. But this conditional destructure seems safe as well.

Reply:
> Would just adding the prop as bodyOpenClassName={ bodyOpenClassName } be just as safe maybe?

I think you're right. I tested this further, and it looks like the Modal control is looking for a literal `null` when it removes the body class name. I'll roll this back to using `bodyOpenClassName={ bodyOpenClassName }` because it's easy to read, and maybe someone might want to send `null` on purpose in the future.